vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.cpp

```diff
@@ -121,6 +121,12 @@ class UIManagerModule : public std::enable_shared_from_this<UIManagerModule>, pu
       vm->GetConstants(writer);
       writer.WriteObjectEnd();
     }
+    auto AccessibilityEventTypes = React::JSValueObject{
```

Comment:
nit, name local variable name starting with lowercase
Also this can be made `const`